addons/payment/tests/test_payment_provider.py

```diff
@@ -226,3 +226,31 @@ def test_provider_not_compatible_with_express_checkout(self):
             self.company.id, self.partner.id, self.amount, is_express_checkout=True
         )
         self.assertNotIn(self.provider, compatible_providers)
+
+    def test_duplicate_provider_child_company_no_journal_id(self):
```

Comment:
The dependency on `account` is in the `account_payment` module; that's where the test should be.

Reply:
Sorry I didn't have the eyes "en face des trous" this day. I'll correct that